_pages/documentation/general_docs/kconfig_build_system.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: documentation
+title: Kconfig Build System
+doc: gem5 documentation
+parent: kconfig_build_system
+permalink: /documentation/general_docs/kconfig_build_system/
+---
+
+This guide is intended for advanced users who need to build GEM5 with
+multiple ISAs or customize the build options, such as the Ruby memory protocol.
+Familiarity with the Kconfig system is required.
+
+## Build GEM5 with Kconfig Build System
+
+```bash
+scons [OPTIONS] Kconfig_command TARGET
+```
+
+Supported kconfig commands include
```

Comment:
Looking at the Kconfig documentation, the capitalization should always be "Kconfig" capital K lowercase config.

I know these seem like nitpicks, but if we don't get them right the first time, we'll never get them right :smile: